src/main/kotlin/it/pagopa/wallet/audit/LoggingEvent.kt

```diff
@@ -16,7 +18,8 @@ data class WalletMigratedAddedEvent(val walletId: String) : LoggingEvent()
 
 data class WalletDeletedEvent(val walletId: String) : LoggingEvent()
 
-data class SessionWalletCreatedEvent(val walletId: String) : LoggingEvent()
+data class SessionWalletCreatedEvent(val walletId: String, val auditWallet: AuditWalletCreated) :
```

Comment:
Since we are adding only one field to SessionWalletCreatedEvent, why creating a new class to encapsulate it and not just adding it to SessionWalletCreatedEvent as a constructor parameter? Like 
`data class SessionWalletCreatedEvent(val walletId: String, val orderId: String)`

Reply:
we want the output in the DB to be something like this with only one key in the auditWallet object, and that would be orderId
![image](https://github.com/user-attachments/assets/619c708c-7761-4755-baa2-d22e41276de2)